pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+requires = ["setuptools", "wheel", "Cython>=0.29.24", "numpy>=1.17.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "scikit-surprise"
+authors = [{ name = "Nicolas Hug", email = "contact@nicolas-hug.com" }]
+description = "An easy-to-use library for recommender systems."
+requires-python = ">=3.8"
+keywords = [ "recommender", "recommendation system" ]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "Intended Audience :: Education",
+  "Intended Audience :: Science/Research",
+  "Topic :: Scientific/Engineering",
+  "License :: OSI Approved :: BSD License",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+]
+license = { "text" = "GPLv3+" }
```

Comment:
```suggestion
license = { "file" = "README.md" }

Reply:
The license is not GPLv3+, so just point it to  the file.

Reply:
Done ✅